specification/cognitiveservices/data-plane/Face/stable/v1.0/Face.json

```diff
@@ -4203,8 +4194,8 @@
       "x-ms-parameter-location": "method",
       "collectionFormat": "csv"
     },
-    "Endpoint": {
-      "name": "Endpoint",
+    "endpoint": {
```

Comment:
Please do not make any changes to other APIs.

Reply:
reverted.